src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+import { getPeople } from '../../api';
+
+import { Person } from '../../types/Person';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isLoad, setIsLoad] = useState<boolean>(false);
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
+    setIsLoad(true);
+    getPeople()
+      .then((peopleFromServer) => setPeople(peopleFromServer))
+      .catch(() => {
+        setError('Something went wrong');
+      })
+      .finally(() => setIsLoad(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {isLoad && <Loader />}
```

Comment:
I think you can use the ternary operator here, because if you have isLoad `true` then the page will show the loader and otherwise the table will be displayed